src/evict/evict_lru.c

```diff
@@ -2268,11 +2269,14 @@ __evict_page(WT_SESSION_IMPL *session, bool is_server)
 	WT_REF *ref;
 	WT_TRACK_OP_DECL;
 	uint64_t time_start, time_stop;
+	uint32_t previous_state;
 	bool app_timer;
 
 	WT_TRACK_OP_INIT(session);
 
-	WT_RET_TRACK(__evict_get_ref(session, is_server, &btree, &ref));
+	previous_state = WT_REF_MEM;
```

Comment:
Since this is an initialization, it should be paired with the `WT_TRACK_OP_INIT` above, not the `__evict_get_ref` call. For cases like this where the initialization isn't required, but a compiler wants it we generally tag them with a comment that says so: `/* -Werror=maybe-uninitialized */` see block_write.c:239 for an example.